### ISSUE TITLE
[Feature Request] Scalable v-avatar text (initials) and icons

### ISSUE THREAD

**Issue by wernerm** [vuetifyjs/vuetify#1835]
The `size` prop enables the scaling of a `v-avatar` component's child `img` or `v-icon` to the specified size. The same does not apply to _text-only_ `v-avatar` content, which is currently implemented as follows:

```
<v-avatar class="teal">
  <span class="white--text headline">C</span>
</v-avatar>
```

Many interfaces fall back to _initial_ (first-letter-text) avatars when an image is not readily available.  Although this is supported in Vuetify (albeit with the font-size predefined by Vuetify's existing typography styles), the scaling is not supported. 
 
### What will it allow you to do that you can't do today?
Scalable text-only `v-avatar` components. Support for double-letter scalable text avatars. Users will be able to defined `-v-avatar` components of any `size`, regardless of whether they contain images, `v-icons` or text.

### How will it make current work-arounds straightforward?
At the moment one has to use a custom component, e.g. an implentation based on  [https://codepen.io/arturheinze/pen/ZGvOMw](https://codepen.io/arturheinze/pen/ZGvOMw)

### What potential bugs and edge cases does it help to avoid?
Fiddling with inline styles and additional scripting to make text scaling work in `v-avatar`.

**Comment by sh7dm** [comment]
Looks like this is already possible:
```vue
<v-avatar
  size="128"
  color="green"
>
  <span class="white--text headline">
    CJ
  </span>
</v-avatar>
```
![default](https://user-images.githubusercontent.com/19504461/52524966-ea1c4480-2cb3-11e9-8075-f77b55331d8f.png)

**Comment by sh7dm** [comment]
Same for icons:
![default](https://user-images.githubusercontent.com/19504461/52536775-0382c680-2d70-11e9-8e79-b65b7d58d31a.png)

**Comment by blalan05** [comment]
This seems to have been implemented in the 2.0 alpha
